### DISCUSSION TITLE
OSCAL 1.0.0 Release

### DISCUSSION THREAD

**Discussion by david-waltermire** [usnistgov/OSCAL#953] [Releases]
The NIST Open Security Controls Assessment Language (OSCAL) team is pleased to announce the release of OSCAL 1.0.0. This first official, major release of OSCAL provides a stable OSCAL 1.0.0 for wide-scale implementation. This release marks an important milestone for the OSCAL project and for the earlier adopters and implementers of security automation with OSCAL.

This release [incorporates changes](https://pages.nist.gov/OSCAL/reference/release-notes/#oscal-100-release) based on feedback from the OSCAL community. The NIST OSCAL team is very thankful for all of the great ideas and feedback we have received to date.

Looking forward, the NIST OSCAL team is excited to work with the [OSCAL community](https://pages.nist.gov/OSCAL/contribute/) to continue to enhance OSCAL through [additional minor releases](https://pages.nist.gov/OSCAL/contribute/roadmap/#beyond-the-oscal-100-full-release).

For additional information on the OSCAL project, please see the NIST’s Cybersecurity Insights blog: [*“The Foundation for Interoperable and Portable Security Automation is Revealed in NIST’s OSCAL Project”*](https://www.nist.gov/blogs/cybersecurity-insights/foundation-interoperable-and-portable-security-automation-revealed) and the [OSCAL website](https://pages.nist.gov/OSCAL).

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/OSCAL/releases/tag/v1.0.0'>OSCAL 1.0.0 Release</a>.</em>

**Comment by bradh** [comment]
Congratulations and much thanks to the core team and everyone involved,

**Reply by david-waltermire** [reply]
@bradh Thanks for all your help along the way!

**Comment by its-a-lisa-at-work** [comment]
Congrats!!

**Comment by rgutwein** [comment]
Congrats!!
